python/social_bot/gazebo_agent.py

```diff
@@ -54,6 +58,13 @@ def __init__(self,
                 if None it will be set the same as agent_type
             config (dict): the configuarations for the agent
                 see `agent_cfg.jason` for details
+            use_pid_control (bool): whether use pid controller for agent.
```

Comment:
Perhaps we don't need this parameter to override default one in agent_cfg.json here. Use PID or not does not changes very often. Editing the "agent_cfg.json" will be fine.

Reply:
Good point.  Let me remove this.